internal/cmdtest/test_cmd.go

```diff
@@ -165,7 +167,7 @@ func (tt *TestCmd) ExpectRegexp(regex string) (*regexp.Regexp, []string) {
 	)
 	tt.withKillTimeout(func() { matches = re.FindReaderSubmatchIndex(rtee) })
 	output := rtee.buf.Bytes()
-	if matches == nil {
+	if matches == nil && !bigCmp(string(output), regex) {
```

Comment:
why do we need to modify this?

Reply:
In order to compare `9.04625697166532776746648320380374280103671755200316906558262375061821325312e+74` with `0x200000000000000000000000000000000000000000000000000000000000000`.